src/orion/plotting/_core.py

```diff
@@ -0,0 +1,85 @@
+import orion.plotting.backend_plotly as backend
```

Comment:
There is no module docstring.

Also `base.py` would be coherent with the rest of the code instead of `_core.py`.